src/sentry/api/serializers/models/organization_dashboard.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+import six
+from sentry.api.serializers import Serializer, register
+from sentry.models import Dashboard
+
+
+@register(Dashboard)
+class DashboardSerializer(Serializer):
+
+    def serialize(self, obj, attrs, user, *args, **kwargs):
+        data = {
+            'id': six.text_type(obj.id),
```

Comment:
Hmm should I turn the id's into strings or is leaving them ints ok?

Reply:
We usually convert ids to strings so probably best to do so.